pool_templates/ezil.json

```diff
@@ -64,7 +64,52 @@
               "cn.ezil.me:25443",
               "cn.ezil.me:35443"
             ]
-          }
+          },
+          {
+            "geo": "Europe-ETH only",
```

Comment:
Could you add spaces around `—`? @R2D2me

Reply:
@Snuff "Europe - ETH only" ?

Reply:
Aha @R2D2me